Applications/ApplicationsLib/ProjectData.cpp

```diff
@@ -995,3 +1002,26 @@ void ProjectData::parseCurves(
             "The curve name is not unique.");
     }
 }
+
+void ProjectData::parseChemicalSystem(
+    boost::optional<BaseLib::ConfigTree> const& config)
+{
+    if (!config)
+        return;
+
+    //! \ogs_file_param{prj__chemical_system__chemical_solver}
+    auto const chemical_solver =
+        config->getConfigAttribute<std::string>("chemical_solver");
+
+    if (boost::iequals(chemical_solver, "Phreeqc"))
+    {
+        DBUG("Configuring phreeqc interface for water chemistry calculation.");
+
+        _chemical_system = ProcessLib::ChemistryLib::createPhreeqcInterface(
+            _mesh_vec[0]->getNumberOfBaseNodes(), config);
+    }
+    else
+        OGS_FATAL(
```

Comment:
be consistent with braces, if an `if` is braced, then the `else` also should be. Use clang-tidy's readability-braces-around-statements check for automatic fix.

Reply:
fixed